sycl/include/sycl/aliases.hpp

```diff
@@ -89,29 +89,37 @@ class half;
 namespace sycl {
 __SYCL_INLINE_VER_NAMESPACE(_V1) {
 using byte __SYCL2020_DEPRECATED("use std::byte instead") = std::uint8_t;
-using schar = signed char;
-using uchar = unsigned char;
-using ushort = unsigned short;
-using uint = unsigned int;
-using ulong = unsigned long;
-using longlong = long long;
-using ulonglong = unsigned long long;
+using schar __SYCL2020_DEPRECATED("") = signed char;
```

Comment:
I'm just curious - why we deprecating them? Is it decided to fully remove these aliases?

Reply:
They don't exist in SYCL 2020 already. I'm not even sure that they should have been in SYCL 1.2.1.

The older spec says in 4.10.2.2 Aliases

> SYCL provides aliases for vec<dataT, numElements> as <dataT><numElements> for the data types: char, short,
int, long, float, double, half, cl_char, cl_uchar, cl_short, cl_ushort, cl_int, cl_uint, cl_long, cl_ulong,
cl_float, cl_double and cl_half and the data types: signed char , unsigned char, unsigned short, unsigned
int, unsigned long, long long and unsigned long long represented with the short hand schar, uchar, ushort,
uint, ulong, longlong and ulonglong respectively, for number of elements: 2, 3, 4, 8, 16. For example the alias
to vec<float, 4> would be float4.

To me that sounds like there should only be `schar4`, for example, but not `schar`.

Reply:
Okay, thanks for the explanation